src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -192,4 +198,15 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
     this.loadActiveEntrySubscription.unsubscribe();
     this.actionSetDateSubscription.unsubscribe();
   }
+
+  requiredFieldsForInternalAppExist(customerName) {
+    const emptyFields = this.entryForm.value.uri === '' && this.entryForm.value.description === '';
+    const isInternalApp = customerName.includes('ioet');
```

Comment:
is this rule skipping those projects? or are need to skip those projects?
- ioet-internal
- other trainning
- safari books
- english lessons
- university

Reply:
we are not skipping those projects, as it is, they should add a description

Reply:
which I think is ok, users should describe.... except from english lessons, there is not really much to describe there